### PR TITLE
Fixed some markdown docs

### PR DESCRIPTION
**Pull request by vsimko**
This pull request contains few fixes in the markdown documentation.
It relates to the Issue #1624

### CODE REVIEW

File: docs/vector/vector-intro.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # geotrellis.vector
 
->"Raster is faster but vector is correcter."
-— Somebody
+> “Yes raster is faster, but raster is vaster and vector just SEEMS more corrector.”
```
Comment (lossyrob) [inline docs/vector/vector-intro.md]: This is actually in the geotrellis.raster intro! Makes it a funny dialog between geotrellis.vector and geotrellis.raster :)

https://github.com/geotrellis/geotrellis/blob/master/docs/raster/raster-intro.md#geotrellisraster

Comment (vsimko) [inline docs/vector/vector-intro.md]: I see :)
While reading both docs, I thought somebody forgot to copy the quotation from raster-intro to vector-intro.

Comment (vsimko) [inline docs/vector/vector-intro.md]: Should I revert the change inside `vector-intro.md` then ?

Comment (fosskers) [inline docs/vector/vector-intro.md]: Yes please.
